app/views/RoomMembersView/components/ActionsSection.tsx

```
@@ -1,111 +1,13 @@
-							})
-						}
-						testID='room-actions-invite-user'
-						left={() => <List.Icon name='user-add' />}
-						showActionIndicator
-					/>
-					<List.Separator />
-				</>
-			) : null}
-		</View>
-	);
+export default function ActionsSection({}: IActionsSection): React.ReactElement | null {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)